build.gradle

```diff
@@ -61,20 +61,22 @@ dependencies {
     implementation "org.koin:koin-ktor:$koin_version"
 
     // Database stuff
-    implementation "com.h2database:h2:1.4.197"
-    implementation "org.postgresql:postgresql:42.2.9"
-    implementation "com.zaxxer:HikariCP:3.4.2"
+    implementation "org.postgresql:postgresql:42.2.17"
+    implementation "com.zaxxer:HikariCP:3.4.5"
     implementation "org.jetbrains.exposed:exposed-core:$exposed_version"
     implementation "org.jetbrains.exposed:exposed-dao:$exposed_version"
     implementation "org.jetbrains.exposed:exposed-jdbc:$exposed_version"
-    implementation "org.flywaydb:flyway-core:6.1.4"
+    implementation "org.flywaydb:flyway-core:7.0.2"
 
     testImplementation "io.ktor:ktor-server-test-host:$ktor_version"
     testImplementation "org.koin:koin-test:$koin_version"
     testImplementation "org.junit.jupiter:junit-jupiter-api:$junit_version"
     testImplementation "org.junit.jupiter:junit-jupiter-params:$junit_version"
-    testImplementation "org.junit.jupiter:junit-jupiter-engine:$junit_version"
+    testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:$junit_version"
     testImplementation "org.amshove.kluent:kluent:1.61"
+    testImplementation "org.testcontainers:testcontainers:1.14.3"
```

Comment:
Added test containers and removed H2 because we don't need to have an in-memory DB anymore :)

Reply:
We don't need to add any configuration for github actions file for test containers to work right? From what I see the tests run without any error

Reply:
For the [outbox-event-bus](https://github.com/waterdog-oss/outbox-event-bus/blob/master/.github/workflows/ci.yml) it was necessary to add the docker service. Not sure here if needed.